agora_backup/agoraBackup.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+#########################################################################################
+# Agora Backup - A simple script that compresses all the necessary files needed for
+# a complete backup.
+#
+# Copyright (C) 2020 GRNET.
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# Along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.   
+#
+# Contributor(s):
+#	* Anastasios Lisgaras <tasos@admin.grnet.gr>
+#########################################################################################
+
+set -u
+set -f
+set -o pipefail
+
+while getopts h:u:p:d: option 
+do
+	case "${option}" in
+	h|--help)
+		echo -e "Agora Backup."
+		echo -e "------------------------------------------------------------------------------------------ "
+		echo -e "agoraBackup.sh [arguments]"
+		echo -e " "
+		echo -e "Arguments:"
+		echo -e "--help \tShow brief help."
+		echo -e "-u \tDatabase username - Specify the username that has rights in the database you want to back up."
+		echo -e "-p \tDatabase password - Specify the database user password."
+		echo -e "-d \tDatabase name\t  - Specify the name of the database for which you want to back up."
+		echo -e "------------------------------------------------------------------------------------------ "
+		echo -e "Usage: ./agoraBackup.sh -u user -p 'password' -d database"
+		exit 0
+		;;
+	u) DATABASE_USER=${OPTARG} ;;
+	p) DATABASE_PASSWORD=${OPTARG} ;;
+	d) DATABASE_NAME=${OPTARG} ;;
+	esac
+done
+
+
+timestamp=$(date +%Y-%m-%d)
+web_server_conf_files="/etc/apache2/"
```

Comment:
do we need all the files from apache2 or the config ones?

Reply:
I kept this setting because I saw that you had it in older script.
Personally, I don't think it's necessary, but it can be useful if we have all the settings from web server.
In addition :

0. Taking all the settings we were sure we have everything and we don't need to select specific vhost settings files.
1. They are not huge files - they do not consume much storage space.